libass/ass_blur.c

```diff
@@ -744,7 +744,7 @@ static void calc_coeff(double mu[4], const int index[4], int prefilter, double r
         (  17 -  126 * mul +  273 * mul2 -  164 * mul3) / 12096,
     };
 
-    double mat_freq[13];
+    double mat_freq[14];
```

Comment:
I've tried to document algorithm [here](https://github.com/MrSmile/CascadeBlur/releases/download/v0.5-alpha/cascade-blur.pdf), but it's too math-heavy. Basically `mat_freq` and `vec_freq` used in calculation of coefficients for least squares problem and any sufficient size should work. I'd tried to determine minimal required size, but forgot that `coeff_blur121` requires additional zero element to work.